packages/backend/src/controllers/v2/SemanticLayerController.ts

```diff
@@ -194,8 +199,8 @@ export class SemanticLayerController extends BaseController {
     async createSemanticViewerChart(
```

Comment:
methods from this one to below should be under `SavedSemanticViewerController.ts` instead